drivers/include/sen5x.h

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (C) 2023 TU Braunschweig Institut für Betriebssysteme und Rechnerverbund
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#ifndef SEN5X_H
+#define SEN5X_H
+
+/**
+ * @defgroup    drivers_sen5x Sensirion Embedded I2C SEN5x Driver
+ * @ingroup     drivers_sensors
+ * @ingroup     drivers_saul
+ * @brief       Driver for I2C communication to SEN5x devices.
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Daniel Prigoshij <prigoshi@ibr.cs.tu-bs.de>
+ */
+
+#include "periph/i2c.h"
+#include <stdint.h>
+#include "saul.h"
```

Comment:
```suggestion
#include <stdbool.h>
#include <stdint.h>

#include "periph/i2c.h"
#include "saul.h"
```

(`stdbool.h` is needed for `bool sen5x_data_ready_flag(const sen5x_t *dev);`, this currently fails on ESP8266 in the CI due to that.)